README.md

```diff
@@ -1,5 +1,3 @@
-[![Build Status](https://travis-ci.org/abo-abo/swiper.svg?branch=master)](https://travis-ci.org/abo-abo/swiper)
-
```

Comment:
Why not replace this with `https://github.com/abo-abo/swiper/actions/workflows/test.yml/badge.svg`?

Reply:
The build status can already be displayed next to the commit on the top page.
![image](https://user-images.githubusercontent.com/2631472/124373289-15ac8900-dccc-11eb-887e-691c1ddef083.png)

I would just remove the redundant badge so even if someone fork the repo, the status displayed still correct.

Reply:
> I would just remove the redundant badge

It's more prominent than commit indications, and has been with us for a while, so I see no reason to remove it now.

> so even if someone fork the repo, the status displayed still correct.

I don't think we need to concern ourselves with forks; they can rely on the commit indication as you suggested.